docs/kqueen.rst

```diff
@@ -152,6 +152,142 @@ in the configuration file, set the environment variable matching the KQUEEN_<con
       - 127.0.0.0/8
       - Addresses allowed to access metrics endpoint without token
 
+Before you provision a Kubernetes cluster, you may need to deploy and configure the following external services:
+
+External services deployment
+-------
+* **Members managing** - To provide user registration by invitations from other users by emails, mail server should be set up.
```

Comment:
To provide user registration by invitations from other users by emails --> To provide user registration by emails invitations from other users.

Reply:
ok